src/CommandingOfficers/AW2/OS/Max.java

```diff
@@ -61,7 +61,7 @@ public Max(GameScenario.GameRules rules)
   {
     super(coInfo, rules);
 
-    CommanderAbility.CostBasis cb = new CommanderAbility.CostBasis(CHARGERATIO_FUNDS);
+    CommanderAbility.CostBasis cb = getGameBasis();
```

Comment:
Since every CO needs to call this, can we just move this now-universal call into the parent constructor?

Reply:
You could also make `getGameBasis()` abstract at the base `Commander` level, and only have one place where this is called.

Reply:
Every CO needs _at least one_ basis. Ave's an example that has multiple.

My initial thought on this was "but only the child has access to the powers at construction time", but we are technically already constructing many powers with a null basis for description purposes.
I see two "easy options" for a single callsite:
1) Parent defines a basis instance variable and fills it out. Child uses instance var.
2) Child COs construct their powers with no basis, and the parent fills them out.

1 is simple and breaks nothing, but it's misleading (The Commander->basis relationship is `1:n`, not `1:1`... even if `n` is usually 1).
2 can not break Ave by doing null checks on the basis first, I guess. But it's multi-stage initialization for something really simple and otherwise-intuitive.

Reply:
I was thinking along the lines of 1), but you're right, Ave is the fly in the soup here. All right, I'm sorry I ruin things, and please carry on.

Reply:
I ruin everything often enough. You need your turn in the spotlight.